### PR TITLE
Bugzilla EDA Tweet

### PR DESCRIPTION
**Pull request by aakankshaduggal**
This PR adds [Bugzilla EDA](https://youtu.be/FMXv3l4NsPc) tweet.

### CODE REVIEW

**Review by oindrillac — CHANGES_REQUESTED** [review]
@aakankshaduggal can you modify this to be under 280 characters?

**Comment by aakankshaduggal** [comment]
Yes @oindrillac 

On it!

File: tweets/2021-07/bugzilla.tweet [hunk 1]
```diff
@@ -0,0 +1 @@
+Analyze your CI processes to learn how bugs are impacting your product. Check out https://youtu.be/FMXv3l4NsPc to learn how to fetch bugs linked to failing tests from Red Hat Bugzilla and perform analysis ?
```
Comment (oindrillac) [inline tweets/2021-07/bugzilla.tweet:1]:
```suggestion
Analyze your CI processes to learn how bugs are impacting your product. Check out https://youtu.be/FMXv3l4NsPc to learn how to fetch bugs linked to failing tests from Red Hat Bugzilla and perform analysis.
```

**Review by oindrillac — CHANGES_REQUESTED** [review]

**Review by oindrillac — COMMENTED** [review]
/lgtm
/hold

**Pull request merged**